### ISSUE TITLE
Add function to user list to turn on/off reporting

### ISSUE THREAD

**Issue by nthieberger** [paradisec-archive/nabu#411]
#410 adds reporting, and we need to add a function to the user database that turns on/off reporting. This then has the effect of automatically allowing or prohibiting reporting. We also need to add a search function for these conditions to the user search screen.

**Comment by enwardy** [comment]
presumably still outstanding, but no indication of priority , @nthieberger ?

**Comment by johnf** [comment]
@nthieberger I don't think this is relevant since I don't think there is any emailing of reports?
